### PR TITLE
Martin/channel invite

### PR DESCRIPTION
**Pull request by tomorrowmartin**
Got basic channel creation and inviting working. You can select a contact and we'll automatically create a channel then invite the contact if possible. Currently, only existing users can be invited.

### CODE REVIEW

File: Source/Managers/ChannelManager.swift [hunk 1]
```diff
@@ -69,7 +69,6 @@ class ChannelManager: NSObject {
     }
 
     static func createChannel(channelName: String,
-                              uniqueName: String,
```
Comment (tomorrowmartin) [inline Source/Managers/ChannelManager.swift]: This parameter is not used. The function automatically generates a UUID.

File: Source/Managers/CollectionViewManager.swift [hunk 2]
```diff
@@ -15,9 +15,6 @@ class CollectionViewManager<CellType: DisplayableCell & UICollectionViewCell>: N
     var collectionView: UICollectionView
 
     var items = MutableProperty<[CellType.ItemType]>([])
-    // A deep copied array representing the last state of the items.
-    // Used to animate changes to the collection view
-    private var previousItems: [CellType.ItemType]?
```
Comment (tomorrowmartin) [inline Source/Managers/CollectionViewManager.swift]: Fixes a crash in the collection view.

File: Source/Models/Data Models/PFUser+Extensions.swift [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        return firstInitial + lastInitial
+    }
+
+    var firstName: String {
+        get {
+            return String(optional: self.object(forKey: "firstName") as? String)
+        }
+        set {
+
+            self.setObject(newValue, forKey: "firstName")
```
Comment (tomorrowmartin) [inline Source/Models/Data Models/PFUser+Extensions.swift]: My generic approach didn't work out. Just going to implement this convenience properties manually.

File: Source/UI/Screens/Home/HomeViewController.swift [hunk 4]
```diff
@@ -142,4 +131,48 @@ class HomeViewController: FullScreenViewController {
...
+                }
+
+                guard let user = objects?.first, let identifier = user.objectId else { return }
+                self.createChannel(with: identifier)
+            })
+        }
+    }
+
+    func createChannel(with inviteePhoneNumber: String) {
+        ChannelManager.createChannel(channelName: "TEST CHANNEL", type: .private)
```
Comment (tomorrowmartin) [inline Source/UI/Screens/Home/HomeViewController.swift]: Channels are all private now.

File: Source/Managers/ChannelManager.swift [hunk 5]
```diff
@@ -139,7 +138,7 @@ class ChannelManager: NSObject {
         guard let members = channel.members else { return }
 
         members.invite(byIdentity: identity) { result in
-
+            print(result)
```
Comment (Benjamindodgson) [inline Source/Managers/ChannelManager.swift]: take out the print

**Pull request merged**